DDR4 OC Guide.md

```diff
@@ -273,7 +273,7 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
   | D | Hynix 8Gb AFR, Hynix 8Gb BFR, Micron 8Gb Rev. B, Micron 8Gb Rev. H, Samsung 8Gb C-Die, Samsung 4Gb D-Die | Low end ICs commonly found in average cheap kits. Most are EOL and no longer relevant. 
   | F | Hynix 8Gb MFR, Micron 4Gb Rev. A, Samsung 4Gb S-Die, Nanya 4Gb A-Die, Nanya 8Gb C-Die | Terrible ICs unable to reliably attain even the highest standard of the base JEDEC Specification.|
   * Partially based off [Buildzoid's older ranking](https://www.reddit.com/r/overclocking/comments/8cjla5/the_best_manufacturerdie_of_ddr_ram_in_order/dxfgd4x/). Some ICs are not included in this list due to age of the post.
-  * <sup>1</sup> 8Gb Rev. E is extremely varied. The chips and various revisions that all fall under 8Gb Rev. E range from an IC that is comfortable overclocking with above average timings to an extremely strong one, being the world record holding IC and being known to get certain timings insanely tight.
+  * <sup>1</sup> 8Gb Rev. E has a slight amount of variation. This mainly manifests as the minimum tRCD and tRP to which the ram scales. Similarly, timings such as tRAS, tRC and tRFC may be affected. VTT is a voltage that requires tuning on Rev. E to clock high and OC far, and different chips of Rev. E may behave differently in regards to how high they can clock without modification of VTT.
```

Comment:
```suggestion
  * <sup>1</sup>Revisions of 8Gb Rev. E mainly differ in the minimum tRCD achievable and how high they can clock without modification of VTT while maintaining stability. Generally, newer revisions of 8Gb Rev. E (C9BKV, C9BLL, etc.) do tighter tRCD and clock higher without modification of VTT.
```

I don't think there's much difference in tRP, tRAS, tRC and tRFC outside of silicon lottery.

Reply:
Sure. I think it's more of a trend of silicon purity having gone up over time. Like how there shouldn't be a fundamental difference between coffee and coffee refresh.